extension/docs/WebComponentsIntegrationGuide.md

```diff
@@ -0,0 +1,650 @@
+
+### Adyen payment refusals
+
+If you receive an HTTP 200 response with an Error or Refused resultCode, check the refusal reason and, if possible, modify your request.
+
+Check the following table to see the mapping of Adyen [result codes](https://docs.adyen.com/development-resources/response-handling#error-codes-types) to CTP [transaction state](https://docs.commercetools.com/http-api-projects-payments#transactionstate)
+|Adyen result code| CTP transaction state
+| --- | --- |
+| Authorised| Success|
+| Refused| Failure|
+| Error| Failure|
+
+### Shopper successfully paid but `redirectUrl` was not reached
+In some payment redirect cases, there might be a valid payment but no order as shopper did not reach the shop's `redirectUrl`.
+For example, after successfully issued payment shopper loses internet connection or accidentally closes the tab.
+In this case [Notification module](../../notification) will receive later a notification with successful content, process it, and update the payment.
+Usage of scheduled [commercetools-payment-to-order-processor](https://github.com/commercetools/commercetools-payment-to-order-processor) job ensures that for every successful payment
+an order can still be asynchronously created.
+
+### Shopper tries to pay a different amount than the actual order amount
+For redirect payments payment amount is bound to `redirectUrl`.
+After redirect and before the actual finalization of the payment at the provider's page, the shopper is still able to change the cart's amount within the second tab.
+If shopper decides to change cart's amount within the second tab and finalize payment within the first tab, then according to payment amount validation an error
+will be shown and order creation declined.
+
+## Test and go live
+After doing the all steps above and testing the payment methods with your test accounts, then when you are ready to go live, you need to do the following steps described as [testing the integration](https://docs.adyen.com/checkout/components-web/#testing-your-integration).
+
+Additionally, follow the official Adyen [integration checklist](https://docs.adyen.com/development-resources/integration-checklist) to ensure you have a complete implementation into Adyen.
+
+# Manual Capture
+By default, payments are captured immediately after authorisation. For payment methods that support separate authorisation and capture, you also have the option to capture the payment later, for example only after the goods have been shipped. This allows you to cancel the payment in case of any issues with the shipment. 
+
+If you need to explicitly request a capture for each payment please follow our [manual capture documentation](./ManualCapture.md).
+
+# Cancel or refund
+If you want to return the funds to your shopper, but are not certain whether the payment has been captured or not, use the Cancel or Refund functionality. 
```

Comment:
what are the steps to follow if the payment has been captured for sure?

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/329/commits/502d2fac7105d8bce95ef463227c31719aecb875